client/src/components/Slider.jsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable import/no-unresolved */
+import { Swiper, SwiperSlide } from "swiper/react";
+import "swiper/css";
+import "swiper/css/navigation";
+import "swiper/css/pagination";
+import "../assets/styles/slider.css";
+import { Navigation, Pagination } from "swiper/modules";
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+
+
+export default function Slider({ sportList }) {
+  return (
+    <Swiper
+      spaceBetween={30}
+      loop
+      navigation
+      pagination={{ clickable: true }}
+      modules={[Navigation, Pagination]}
+      breakpoints={{
+        320: {
+          slidesPerView: 2,
+          spaceBetween: 10,
+        },
+        768: {
+          slidesPerView: 3,
+          spaceBetween: 20,
+        },
+        1024: {
+          slidesPerView: 4,
+          spaceBetween: 30,
+        },
+      }}
+    >
+      {sportList.map((sport) => (
+        <SwiperSlide key={sport.name}>
+          <Link to="/category">
+            <img
+              src={sport.imgSrc}
+              alt={sport.name}
+              className="img-shadow w-[calc(100%-15px)] mr-auto rounded-xl mb-4 h-60 object-cover"
+            />
+          </Link>
+          <p className="flex justify-center items-center font-bold">
```

Comment:
It's strange not to be able to click on the text but only on the image. Something to think about